packages/website/docs/Checkbox.md

```diff
@@ -1,29 +1,57 @@
-## Guidelines
+# Best Practices
 
-### General
+Use checkboxes to allow user to select multiple options or none of them.
 
--   Whereas checkboxes can be used to toggle options on and off, using a [toggle switch](#/components/FlatSelect) or [radio buttons](#/components/Radio) is preferable.
--   A checkbox set should always have a title.
--   Always put the most common options at the top of the checkbox list.
--   If more than 7 options (checkboxes) are needed, it is preferable to group checkboxes logically under more than one set of options such that the set label represents the grouping logic.
+Use a stand-alone checkbox to toggle a single option on or off and when the effect is not immediate.
 
-### Label
+-   E.g. if a final confirmation is required to apply the effect.
 
--   Try to limit labels to one short line.
--   Add instructions that indicate how many items can be selected, as some users might not know the difference between checkboxes and radio buttons.
--   Consider listing options vertically, as horizontal listings can make it difficult to tell which label is related to which checkbox.
--   [Partially selected states](#/components/Checkbox) should only be used when sub-options can be selected as children of a checkbox. A parent checkbox being in the partially selected state indicates that only some of its sub-options are selected, not all of them. Toggling the parent checkbox toggles all its sub-options.
+Try to limit the number of available options under 7. If impossible, consider breaking them down into smaller logical sets or use the [Multi-Select](#/form/MultiSelect)
 
-### Behavior
+Options should be ordered by most relevant unless a more suited ordering rational applies (such as for sizes or security levels). If no rational particularly stands-out, use alphanumerical order.
 
--   As much as possible, preselect the recommended option. If there is none, preselect the option most likely to be selected by users.
+## Labeling
```

Comment:
Same question as in my comments on the RadioButtons PR: should we change this to `Text` since we have `Labels` below?

Reply:
I wouldn't because we have the _Help text and Instruction_ section, also that labeling section is meant to only refer to the Titles and options labels which are two type of "labels" that make your information architecture tangible and actionable for your users.
Let's keep labeling